src/main/java/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTService.java

```diff
@@ -475,6 +478,15 @@ public List<RESTGekoppeldeZaakEnkelvoudigInformatieObject> listGekoppeldeZaakInf
         return enkelvoudigInformatieobjectList;
     }
 
+    @POST
+    @Path("/informatieobject/{uuid}/ondertekenen")
+    public Response signDocument(@PathParam("uuid") final UUID uuid, @QueryParam("zaak") final UUID zaakUUID) {
```

Comment:
We houden voor de rest services de confentie dat de methode namen hetzelfde zijn in front en backend. 
In de frontend heb je de methode: ondertekenInformatieObject genoemd, graag hier dus ook dezelfde naam.

Reply:
Hernoemd in nieuwe commit